schemas/questionnaire_v1.json

```
@@ -402,6 +402,10 @@
         },
         "required": ["id", "groups"]
       }
+    },
+    "preview_questions": {
+      "type": "boolean",
+      "description": "A flag that is used to determine if the questions are previewed before the questionnaire start"
```

Why did we need an introduction page?. If it is specific to introduction page, then we should make this flag a property of Introduction block.

```suggestion
      "description": "A flag that is used to determine if the questions can be previewed before the questionnaire start"
```